vortex-sampling-compressor/src/compressors/alp_rd.rs

```diff
@@ -27,6 +27,10 @@ impl EncodingCompressor for ALPRDCompressor {
         ALPRD::ID.as_ref()
     }
 
+    fn decompression_seconds_per_gb(&self) -> f64 {
+        0.39
```

Comment:
Can you maybe make this constants in the file so it's easier to know what to tune?